lib/internal/repl/import.js

```diff
@@ -2,6 +2,19 @@
 
 const acorn = require('internal/deps/acorn/dist/acorn');
 
+/* transforms
+"import { x, y as z } from 'z'; x(); z + 1"
+
+"(async () => {
```

Comment:
Where is `x` and `z` initialized? Can you add that to the transformed source example.

Reply:
they aren't, like the await transform i rely on them attaching top scope